### PR TITLE
Use fixed frame for `TextStrut`

### PR DESCRIPTION
**Pull request by PavelHolec**
With this change, the `TextStrut` (a component that maintains some minimal text-size driven height for other components) gets a consistent size, independent on currently used font. A benefit is that the components are now more stable in size and match the Orbit definition (for the default, large, dynamic type).

Side effect is that this will require most snapshots to be updated.

image

### CODE REVIEW

**Comment by PavelHolec** [comment]
Sizes need to be rechecked. Using `CircularPro` should match orbit heights in normal font sizes.

File: Sources/Orbit/Components/Badge.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ import SwiftUI
 /// - Note: [Orbit definition](https://orbit.kiwi/components/badge/)
 public struct Badge: View {
 
-    public static let verticalPadding: CGFloat = 4 + 1/3 // Results in ±24 height at normal text size
+    public static let verticalPadding: CGFloat = 4 // Results in ±24 height at normal text size
```
Comment (sjavora) [inline Sources/Orbit/Components/Badge.swift:11]: `.xxSmall`?

**Review by sjavora — APPROVED** [review]

File: Sources/Orbit/Components/Badge.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/Badge.swift]: Fixed 👍

**Review by sjavora — APPROVED** [review]

**Comment by PavelHolec** [comment]
Will be replaced by `Strut` with simpler calculation

**Pull request closed without merging**